### PR TITLE
Internships

### PR DESCRIPTION
**Pull request by rdvhoorn**

### CODE REVIEW

File: src/app/core/services/auth.service.ts [hunk 1]
```diff
@@ -33,43 +21,14 @@ export class AuthService {
                 alert(err.error.data);
                 return of();
             }))
-            .subscribe(response => {
-                this.getProfile().then(user => {
-                    return;
-                });
-
+            .subscribe((_) => {
```
Comment (HiddeKoerts) [inline src/app/core/services/auth.service.ts:24]: Why is that underscore there?

File: src/app/core/snippets/footer/footer.component.html [hunk 2]
```diff
@@ -33,14 +33,14 @@ <h5 class="hsa">Honors Study Association Confluente</h5>
...
+          <a href="https://www.linkedin.com/company/h.s.a.-confluente/">LinkedIn</a><br>
           <a
             href="https://calendar.google.com/calendar/embed?src=hsaconfluente%40hsaconfluente.nl&ctz=Europe%2FAmsterdam">Calendar</a>
         </div>
 
         <div class="col-xs-12 col-md-6 text-right">
-          <p>This site is maintained by the <span class="hsa">H.S.A Confluente</span> <a
-            href="mailto:web@hsaconfluente.nl">Web Committee</a>.<br>
+          <p>This site is maintained by the <span class="hsa">H.S.A Confluente</span><a
+            href="mailto:web@hsaconfluente.nl"> Web Committee</a>.<br>
```
Comment (HiddeKoerts) [inline src/app/core/snippets/footer/footer.component.html:43]: Why this change? Older formatting seems more logical?

File: src/app/core/templates/activities/activity-create/activity-create.component.ts [hunk 3]
```diff
@@ -103,29 +107,37 @@ export class ActivityCreateComponent implements OnInit {
...
         }
 
         const fd = new FormData();
         if (hasCoverImage) {
-            // @ts-ignore
-            const file = $('#activityCreate-cover')[0].files[0];
+            const file = this.uploadForm.get('image').value;
+
             if (!file.type.startsWith('image/')) {
                 return this.wrongInput('Non-image formats are not supported as pictures for activities!');
```
Comment (HiddeKoerts) [inline src/app/core/templates/activities/activity-create/activity-create.component.ts:142]: Should we also incorporate a check here for the file type, or is that redundant?

File: src/app/core/templates/activities/activity-create/activity-create.component.ts [hunk 4]
```diff
@@ -203,13 +219,19 @@ export class ActivityCreateComponent implements OnInit {
                     this.empty = true;
                 }
 
-                if (dataObj.fullQuestion.includes("#,#")) { this.wrongCharacters = true; }
-                if (dataObj.fullQuestion.includes("#;#")) { this.wrongCharacters = true; }
+                if (dataObj.fullQuestion.includes("#,#")) {
+                    this.wrongCharacters = true;
+                }
+                if (dataObj.fullQuestion.includes("#;#")) {
+                    this.wrongCharacters = true;
```
Comment (HiddeKoerts) [inline src/app/core/templates/activities/activity-create/activity-create.component.ts:226]: Is this variable ever used? I don't see an error being thrown.

File: src/app/core/templates/partners/internships/internship-overview/internship-overview.component.spec.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (HiddeKoerts) [inline src/app/core/templates/partners/internships/internship-overview/internship-overview.component.spec.ts:1]: This file has different identation levels compared to all others.

File: src/environments/environment.ts [hunk 6]
```diff
@@ -3,7 +3,8 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+    production: false,
+    ROOT_URL: "http://localhost:81",
 };
```
Comment (HiddeKoerts) [inline src/environments/environment.ts:8]: Shouldn't this information only be in the ''environment.prod.ts'' file and its example?

File: src/app/core/services/auth.service.ts (hunk 1 above)
Comment (HiddeKoerts) [reply src/app/core/services/auth.service.ts:24]: (just curious)

File: src/app/core/pages/home/home.component.html [hunk 7]
```diff
@@ -50,13 +50,15 @@
...
-                <li class="bw"><a href="/partners/sponsors"><img class="img-logo" alt="CQM"
-                                                                 src="./assets/img/logo_cqm.png"></a>
-                </li>
-                <li class="bw"><a href="/partners/sponsors/tno"><img class="img-logo" alt="TNO" id="tno-logo"
-                                                                     src="./assets/img/logo_tno.png"></a></li>
+<!--                <li class="bw"><a href="/partners/sponsors"><img class="img-logo" alt="CQM"-->
+<!--                                                                 src="./assets/img/logo_cqm.png"></a>-->
+<!--                </li>-->
+<!--                <li class="bw"><a href="/partners/sponsors/tno"><img class="img-logo" alt="TNO" id="tno-logo"-->
+<!--                                                                     src="./assets/img/logo_tno.png"></a></li>-->
```
[the remark this answers is not in the document]
Comment (rdvhoorn) [reply src/app/core/pages/home/home.component.html:57]: they might come back :p

File: src/app/core/services/auth.service.ts (hunk 1 above)
Comment (rdvhoorn) [reply src/app/core/services/auth.service.ts:24]: catching but not using

File: src/app/core/snippets/footer/footer.component.html (hunk 2 above)
Comment (rdvhoorn) [reply src/app/core/snippets/footer/footer.component.html:43]: nope, the original version did not actually place a space :P

File: src/app/core/templates/activities/activity-create/activity-create.component.ts (hunk 4 above)
Comment (rdvhoorn) [reply src/app/core/templates/activities/activity-create/activity-create.component.ts:226]: yeah it is further down

**Review by HiddeKoerts — APPROVED** [review]
